### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by nrdxp** [divnix/std#201]
We need to cut a release. We have divnix/std-action now and we need to provide some level of interoperability between the two. We can't do that to any level of sanity without tracking a stable target. It doesn't have to be 1.0 but we need to cut something.

**Comment by blaggacao** [comment]
I'm clandestinely waiting for https://github.com/jmgilman/cocogitto-pr, which is clandestinely waiting for `std-action` in turn :smiley: .

But as soon as that's in place, we should be able to do this rather swiftly.

**Comment by blaggacao** [comment]
Blocked by #123

**Comment by nrdxp** [comment]
Its not a hard dependency though right? The cocogitto PR just adds a nice preview of the changelog. Not sure why that would block a release.

**Comment by blaggacao** [comment]
> Its not a hard dependency though right?

No, deliberate. To properly exercise the process. It also puts time bounds on the cocogitto-pr milestone.

**Comment by jmgilman** [comment]
I agree it shouldn't be a hard dependency. It's possible to integrate cocogitto without the existence of the PR generation, which is really just a nicety in the end.

**Comment by blaggacao** [comment]
Another aspect, I didn't mention, but had in mind is to land a release of `std` together with 1.0 releases of the satellite repos (`nosys` / `data-merge` / `yants` / `incl` / `paisano`).

If we package this properly, it can be a true milestone for the project (not just a release).

**Comment by blaggacao** [comment]
Let me actually transform this into a milestone via Github Settings. Closing here.
